.bazelrc

```diff
@@ -29,3 +29,45 @@ run --incompatible_strict_action_env
 build --define kafka_enabled=false
 test --define kafka_enabled=false
 run --define kafka_enabled=false
+
+
+# toolchain build debug config
+build:debug --sandbox_debug
+build:debug --toolchain_resolution_debug
+build:debug --verbose_failures
+
+# linux_amd64 conifg for cross compiler toolchain, not strictly necessary since host/exec env is amd64
+build:linux_amd64 --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64_cgo
+
+# linux_amd64_debug
+build:linux_amd64_debug --config=linux_amd64
+build:linux_amd64_debug --config=debug
+
+# linux_amd64 docker sandbox build config
+build:linux_amd64_docker --config=linux_amd64_debug
+build:linux_amd64_docker --config=docker-sandbox
+
+# linux_arm64 conifg for cross compiler toolchain
+build:linux_arm64 --crosstool_top=//tools/cross-toolchain:multiarch_toolchain
+build:linux_arm64 --platforms=@io_bazel_rules_go//go/toolchain:linux_arm64_cgo
+build:linux_arm64 --copt=-funsafe-math-optimizations
+build:linux_arm64 --copt=-ftree-vectorize
+build:linux_arm64 --copt=-fomit-frame-pointer
+build:linux_arm64 --cpu=aarch64
+build:linux_arm64 --compiler=clang
+build:linux_arm64 --copt=-march=armv8-a
+
+#linux_arm64_debug
+build:linux_arm64_debug --config=linux_arm64
+build:linux_arm64_debug --config=debug
+
+# linux_arm64 docker sandbox build config
+build:linux_arm64_docker --config=linux_arm64_debug
+build:linux_arm64_docker --config=docker-sandbox
+
+# Docker Sandbox Mode - Note all docker sandbox configs must run from a linux x86_64 host
+build:docker-sandbox --experimental_docker_image=suburbandad/cross-clang-10:latest
```

Comment:
we may not want to merge this change since it refers to a non-prysm private repo.  If we want to offer a docker-sandbox target, it should refer to a prysmatic docker repo